active-directory-wpf-msgraph-v2/App.xaml.cs

```diff
@@ -24,11 +24,11 @@ static App()
         //   - for any Work or School accounts, use organizations
         //   - for any Work or School accounts, or Microsoft personal account, use common
         //   - for Microsoft Personal account, use consumers
-        private static string ClientId = "0b8b0665-bc13-4fdc-bd72-e0227b9fc011";
+        private static string ClientId = "Enter_the_Application_Id_here";
 
         // Note: Tenant is important for the quickstart. We'd need to check with Andre/Portal if we
         // want to change to the AadAuthorityAudience.
-        private static string Tenant = "common";
+        private static string Tenant = "Enter_the_Tenant_Info_Here";
```

Comment:
On this one, I would prefer that we keep "common", as otherwise the sample won't work as is. I could change the code, though to replace "Enter_the_Tenant_Info_Here" by "common". Please tell me what you prefer @archieag